APP_README.md

```diff
@@ -0,0 +1,91 @@
+`.azuredevops/` | Used to store the pull request template used by Azure DevOps.
+`build/` | Regroups all yaml files used that compose the pipelines defined in `.azure-pipelines.yml`.
+`build/gitversion-config.yml` | Contains the configuration for the GitVersion tool that is used to compute the version number in the pipelines.
+`.azure-pipelines.yml` | Defines the main CI/CD pipeline of the project.
+`.azure-pipelines-canary.yml` | Defines the canary pipeline of the project.<br/>Canary pipelines allow to detect regressions by periodically creating versions of the app in which all the dependencies are updated to their latest version.
+`doc/` | Regroups all the documentation of the project, with the only exception of `README.md`, which is located at the root of the repository.<br/><br/>**There are many topics covered. Make sure you check them out.**
+`README.md` | The entry point of this project's documentation.
+`src/` | Contains all the code of the application, including tests, with the exception of the configuration files located at the root of the repository.
+`src/{YourProjectName}.sln` | The solution file to use with Visual Studio to develop locally.
+`.editorconfig`<br/>& `stylecop.json` | Configure the code formatting rules and analyzers of Visual Studio.
+`.gitignore` | Contains the git ignore rules.
+`Directory.Build.props` | Regroups build configuration that apply to all `.csproj`.
+`nuget.config` | Contains the configuration for NuGet packages.
+`tools/` | Offers a place to put custom tools and scripts. It also contains some information about the version of the template that was used to generate the project.
+
+## Software Architecture
+
+The software architecture of the application is documented in the [Architecture.md](doc/Architecture.md) document.
+It covers things like the application context, the functional overview, the application structure, the solution structure, the recipes, and more.
+
+## Pipelines
+
+This project uses Azure Pipelines to automate some processes.
+They are described in details in [AzurePipelines.md](doc/AzurePipelines.md).
+
+### Pipelines Summary
+TODO: Fill the following table with your own pipelines.
+
+| Link | Code Entry Point | Goal | Triggers |
+|-|-|-|-|
+| [Name of Main Pipeline](link-to-pipeline)| [`.azure-pipelines.yml`](.azure-pipelines.yml)| Build validation during pull request.| Pull requests.
+| [Name of Main Pipeline](link-to-pipeline)| [`.azure-pipelines.yml`](.azure-pipelines.yml)| Build and deploy the application to AppCenter, TestFlight, and GooglePlay. | Changes on the `main` branch.<br/>Manual trigger.
+| [Name of API Integration Tests Pipeline](link-to-pipeline)| [`.azure-pipelines-api-integration-tests.yml`](.azure-pipelines.yml)| Run all tests, including APIs integration tests. | Daily cron job.<br/>Manual trigger.
+| [Name of Canary Merge Pipeline](link-to-pipeline)| [`build/canary-merge.yml`](.azure-pipelines.yml)| Creation of canary branches (`canary/build/*`). | Daily cron job.
```

Comment:
we may want to remove canary rows here since I don't think we plan to have canaries for flutter projects